src/components/common/incomplete-tests-banner/incomplete-tests-banner.selector.ts

```diff
@@ -0,0 +1,28 @@
+import { StoreModel } from '../../../shared/models/store.model';
+import * as testsSelectors from '../../../modules/tests/tests.selector';
+import { getStartableSlotIdsBeforeToday } from '../../../pages/journal/journal.selector';
+import { DateTime } from '../../../shared/helpers/date-time';
+import { SlotProvider } from '../../../providers/slot/slot';
+
+export const getIncompleteTestsCount = (store: StoreModel, today: DateTime, slotProvider: SlotProvider): number => {
```

Comment:
This selector is not too easy to understand, so I would suggest to add a general comment about the strategy of the counting algorithm.

Reply:
I've refactored and added some comments 👍